eslint/babel-eslint-parser/src/convert/convertAST.js

```diff
@@ -91,11 +91,19 @@ function convertNodes(ast, code) {
     "typeParameters",
   ];
 
+  // Make sure we visit `exported` key to remove `identifierName` from loc node
+  const oldExportAllDeclarationKeys = t.VISITOR_KEYS.ExportAllDeclaration;
```

Comment:
We should add `ExportAllDeclaration` to `eslint/babel-eslint-parser/src/visitor-keys.js`, too.

Reply:
I don't think we need to: https://github.com/eslint/eslint-visitor-keys/blob/master/lib/visitor-keys.json#L71?

Reply:
Well only some of them are picked:

https://github.com/babel/babel/blob/ffb42488ba1a9d05a41d028cd72ae8c77dfe2071/eslint/babel-eslint-parser/src/visitor-keys.js#L7-L13

Reply:
Yeah, guess we have to add back ESLint's version since Babel's will overwrite it